README.md

```diff
@@ -49,7 +56,10 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+# remove Belle from the town residents
+belle = town[:residents].slice!(town[:residents].index("Belle"))
+# add her to the list of guests in the castle.
+town[:castle][:guests].push(belle)
```

Comment:
This works.  However, if something feels complex in ruby then it's time to look for another way.

```
belle = town[:residents].delete("Belle")
town[:castle][:guests] << belle
```